docs/source/release_notes/release-notes.rst

```diff
@@ -45,8 +45,30 @@ develop
          - Change
 
     *    - |improved|
+         - ``getRange`` is now more efficient when scanning over rows with many updates in Cassandra, if just a single column is requested.
+           Previously, a range request in Cassandra would always retrieve all columns and all historical versions of each column, regardless of which columns were requested.
+           Now, we only request the latest version of the specific column requested, if only one column is requested. Requesting multiple columns still results in the previous behavior, however this will also be optimized in a future release.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2480>`__)
+
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+v0.61.0
+=======
+
+18 October 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    - |improved|
          - Sweep is now more efficient on Cassandra, Postgres and Oracle.
```

Comment:
We should be more specific - perhaps we can use the numbers from the benchmarks, at least for DbKvs?

Reply:
```
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanConservative-CASSANDRA) changed SIGNIFICANTLY (-98.32%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanThorough-CASSANDRA) changed SIGNIFICANTLY (-98.35%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyConservative-CASSANDRA) changed SIGNIFICANTLY (-64.94%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyThorough-CASSANDRA) changed SIGNIFICANTLY (-84.33%)  

(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanConservative-ORACLE) changed SIGNIFICANTLY (-81.46%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanThorough-ORACLE) changed SIGNIFICANTLY (-83.86%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyConservative-ORACLE) changed SIGNIFICANTLY (-72.36%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyThorough-ORACLE) changed SIGNIFICANTLY (-69.82%)  

(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanConservative-POSTGRES) changed SIGNIFICANTLY (-53.35%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanCleanThorough-POSTGRES) changed SIGNIFICANTLY (-38.80%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyConservative-POSTGRES) changed SIGNIFICANTLY (-41.30%)  
(benchmark=KvsGetCandidateCellsForSweepingBenchmarks#fullTableScanDirtyThorough-POSTGRES) changed SIGNIFICANTLY (-51.97%)  
```

Reply:
Should I say improved by an order of magnitud? Not sure what to say here as the improvement will be heavily dependent on the size of values, number of versions etc.?